test/integration/highlight_editor_spec.mjs

```diff
@@ -696,8 +696,12 @@ describe("Highlight Editor", () => {
           for (let i = 0; i < 6; i++) {
             await page.keyboard.press("ArrowRight");
           }
-          await page.keyboard.press("ArrowDown");
-          await page.keyboard.press("ArrowDown");
+          // Here and elsewhere, we add a small delay between press and release
+          // to make sure that a keyup event is triggered after
+          // selectionchange.
+          // It works with a value of 10ms, but we use 100ms to be sure.
+          await page.keyboard.press("ArrowDown", { delay: 100 });
```

Comment:
*Most likely I don't fully understand these tests, but:*

What's "special" about ArrayDown here, since just above here (in the for-loop) there's ArrowRight that we apparently don't have to delay?